python/snewpy/flux.py

```diff
@@ -265,10 +265,10 @@ def integrate(self, axis:Union[Axes,str], limits:np.ndarray=None)->'Container':
         if limits is None:
             limits = u.Quantity([xmin, xmax])
         limits = limits.to(ax.unit)
-        limits = limits.clip(xmin, xmax)
+        #limits = limits.clip(xmin,xmax)
         #compute the integral
         yc = cumulative_trapezoid(self.array, x=ax, axis=axis, initial=0)
-        _integral = interp1d(x=ax, y=yc, fill_value=0, axis=axis, bounds_error=False)
+        _integral = interp1d(x=ax, y=yc, fill_value='extrapolate', axis=axis, bounds_error=False)
```

Comment:
I think it is better to make the calculation only in the time window where the initial model flux file is computed and avoid extrapolation, but this should have negligible impact